web/static.go

```diff
@@ -121,3 +139,28 @@ func unsupportedBrowserScriptHandler(w http.ResponseWriter, r *http.Request) {
 	templatesDir, _ := templates.GetTemplateDirectory()
 	http.ServeFile(w, r, filepath.Join(templatesDir, "unsupported_browser.js"))
 }
+
+func getOpenGraphMetaTags(c *Context) string {
+	siteName := model.TeamSettingsDefaultSiteName
+	customSiteName := c.App.Srv().Config().TeamSettings.SiteName
+	if customSiteName != nil && *customSiteName != "" {
+		siteName = *customSiteName
+	}
+
+	siteDescription := model.TeamSettingsDefaultCustomDescriptionText
+	customSiteDescription := c.App.Srv().Config().TeamSettings.CustomDescriptionText
+	if customSiteDescription != nil && *customSiteDescription != "" {
+		siteDescription = *customSiteDescription
+	}
+
+	titleTemplate := "<title>%s</title>"
+	titleHTML := fmt.Sprintf(titleTemplate, html.EscapeString(siteName))
+
+	if siteDescription != "" {
+		descriptionTemplate := "<meta property=\"og:description\" content=\"%s\" />"
+		descriptionHTML := fmt.Sprintf(descriptionTemplate, html.EscapeString(siteDescription))
+		return fmt.Sprintf("%s%s", titleHTML, descriptionHTML)
+	}
```

Comment:
Having one return statement makes the function a bit easier to read

```suggestion
	descriptionHTML := ""
	if siteDescription != "" {
		descriptionTemplate := "<meta property=\"og:description\" content=\"%s\" />"
		descriptionHTML := fmt.Sprintf(descriptionTemplate, html.EscapeString(siteDescription))
	}
	
	return fmt.Sprintf("%s%s", titleHTML, descriptionHTML)
```